mlflow/langchain/__init__.py

```diff
@@ -863,16 +856,21 @@ def _load_pyfunc(path: str, model_config: Optional[Dict[str, Any]] = None):
 
 def _load_model_from_local_fs(local_model_path):
     flavor_conf = _get_flavor_configuration(model_path=local_model_path, flavor_name=FLAVOR_NAME)
-    if MODEL_CODE_PATH in flavor_conf:
-        model_config = flavor_conf.get(MODEL_CONFIG, None)
+    pyfunc_flavor_conf = _get_flavor_configuration(
+        model_path=local_model_path, flavor_name=PYFUNC_FLAVOR_NAME
+    )
+    if MODEL_CODE_PATH in pyfunc_flavor_conf or MODEL_CODE_PATH in flavor_conf:
```

Comment:
can we add a comment to the code explaining why we check both (i.e. flavor_conf is just for backcompat)